docs/reference/mapping/types/core-types.asciidoc

```diff
@@ -150,7 +150,7 @@ limit to `32766 / 3 = 10922` since UTF-8 characters may occupy at most 3
 bytes.
 
 |`position_offset_gap` |Position increment gap between field instances
-with the same field name. Defaults to 0.
+with the same field name. Defaults to the default of the analyzer which is 10.
```

Comment:
Yeah! I'll do that.

10 is totally arbitrary. Its much more than 1 which is too low. I'd be happy to change it to 1024 or some other magic number. I just want it much much more than 1.